### PR TITLE
Release snaplogic-snaplex 4.14.0

### PR DESCRIPTION
**Pull request by VictorDu**
Latest snaplogic-snaplex release

### CODE REVIEW

**Comment by joerg84** [comment]
Just started the CI, will take a look once done

File: repo/packages/S/snaplogic-snaplex/0/package.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "packagingVersion": "2.0",
```
Comment (ryadav88) [inline repo/packages/S/snaplogic-snaplex/0/package.json:2]: change to `4.0`

File: repo/packages/S/snaplogic-snaplex/0/package.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "description": "Snaplogic Snaplex",
+    "maintainer": "support@snaplogic.com",
+    "framework": false,
+    "name": "snaplogic-snaplex",
+    "postInstallNotes": "Snaplogic Snaplex has been installed.",
```
Comment (ryadav88) [inline repo/packages/S/snaplogic-snaplex/0/package.json:7]: add `preInstall` notes

Comment (ryadav88) [reply repo/packages/S/snaplogic-snaplex/0/package.json:7]: create a documentation example and add a link at the end of preInstallNotes

File: repo/packages/S/snaplogic-snaplex/0/package.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    "postUninstallNotes": "Snaplogic Snaplex has been uninstalled. Note that any data persisted to a NFS share still exists and will need to be manually removed.",
+    "selected": true,
+    "tags": [
+      "data-integration",
+      "JCC",
+      "Snaplogic"
+    ],
+    "version": "latest",
```
Comment (ryadav88) [inline repo/packages/S/snaplogic-snaplex/0/package.json:15]: please use version of the format: integrationVersion-baseTechVersion. Ex: 0.1.0-1.2.3 if 1.2.3 is Snaplogic version

File: repo/packages/S/snaplogic-snaplex/0/package.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
+{
+    "packagingVersion": "2.0",
+    "description": "Snaplogic Snaplex",
```
Comment (ryadav88) [inline repo/packages/S/snaplogic-snaplex/0/package.json:3]: add `license`

File: repo/packages/S/snaplogic-snaplex/0/resource.json [hunk 5]
```diff
@@ -0,0 +1,14 @@
+{
+  "assets": {
+    "container": {
+      "docker": {
+        "agent": "snaplogic/snaplex:latest"
```
Comment (ryadav88) [inline repo/packages/S/snaplogic-snaplex/0/resource.json:5]: use a tagged version of an image

**Review by ryadav88 — CHANGES_REQUESTED** [review]
Please also create a documentation example by submitting a PR to https://github.com/dcos/examples
You can use other examples there as a reference.

**Pull request closed without merging**